samples/extensions/shader_debugprintf/shader_debugprintf.cpp

```diff
@@ -84,10 +71,25 @@ ShaderDebugPrintf::~ShaderDebugPrintf()
 
 void ShaderDebugPrintf::request_gpu_features(vkb::PhysicalDevice &gpu)
 {
+	auto const &supportedFeatures = gpu.get_features();
+	auto       &requestedFeatures = gpu.get_mutable_requested_features();
+
+	// debugPrintfEXT requires fragmentStoresAndAtomics, vertexPipelineStoresAndAtomics, and shaderInt64
+	if (supportedFeatures.fragmentStoresAndAtomics && supportedFeatures.vertexPipelineStoresAndAtomics && supportedFeatures.shaderInt64)
+	{
+		requestedFeatures.fragmentStoresAndAtomics       = VK_TRUE;
+		requestedFeatures.vertexPipelineStoresAndAtomics = VK_TRUE;
+		requestedFeatures.shaderInt64                    = VK_TRUE;
```

Comment:
Does this sample really require shaderInt64? My hardware doesn't support it, but if I remove this line it appears to run ok anyway.

Reply:
@gary-sweet thanks for your comments.  I will fix the debugPrintfEXT format in the shader.  Regarding the requested features, I could simply remove them all and have the validation layer implicitly request them as needed.  The reason I added these was to reduce the log clutter as I thought these were the mandatory ones.  However, with recent GPU-AV changes in the VVL, there may acutally be even more features implicitly requested by the VVL.  To help with this decision, can you tell me what platform are you using?  And would you mind turning off all requested features and providing a log of the VVL output showing the implicitly requested features for your machine?

Reply:
Thanks @spencer-lunarg.  Can you tell me which features are now mandatory?

Reply:
1. `fragmentStoresAndAtomics`/`vertexPipelineStoresAndAtomics`
2. `bufferDeviceAddress`
3. `timelineSemaphore`

Reply:
Thanks again.  To keep things simple, I will remove `requestedFeatures.shaderInt64` from this sample and let the VVL implicitly request `bufferDeviceAddress` and `timelineSemaphore`.

Reply:
>  To help with this decision, can you tell me what platform and SDK version are you using? 

That won't help you very much as it's our own Broadcom hardware I'm afraid. SDK looks like 1.4.305

> And would you mind turning off all requested features and providing a log of the VVL output showing the implicitly requested features for your machine?

Do you still need this given Spencer's reply?

Reply:
@gary-sweet I should be good to go as is.  Thanks for the feedback.  Hopefully removing `shaderInt64` will help.